pkg/operator/workloads/api_workload.go

```diff
@@ -250,6 +258,19 @@ func tfAPISpec(
 		tfServingLimitsList["nvidia.com/gpu"] = *kresource.NewQuantity(api.Compute.GPU, kresource.DecimalSI)
 	}
 
+	downloadArgs := []downloadContainerArg{
+		{
+			From: ctx.APIs[api.Name].Model,
+			To:   path.Join(consts.EmptyDirMountPath, "model"),
```

Comment:
What do you think about putting the unzip flag in the struct, in case there's a situation when only a subset should unzip?